tools/assetpipelineeditor/mainwindow.cpp

```diff
@@ -337,6 +345,42 @@ void MainWindow::openFile()
     setFilePath(filePath);
 }
 
+void MainWindow::compressFile()
```

Comment:
need error reporting if compressing fails, write files fails, etc

Reply:
I've added error reporting in the export gui, is it enough ?

Reply:
I guess it'll do for now